### PR TITLE
features-email

### PR DESCRIPTION
**Pull request by shauryatripaathi**
### Changes

- Created new branch named features-email.
  Updated Contact file :-
- Added multiple input fields and submit button in Contact file
- Added style.css to input box.
- Added useState hook in Contact file
- Added onChange event to multiple input fields. 
- Added onSubmit event to form element.
- Updated Project file , added description of Richmond Croft Projects .
- Placeholder added in input fields.
-Update Button Color Changed , Border Color Changed , Button size changed

Screenshot (25)

### CODE REVIEW

File: src/Header/Contact/index.js [hunk 1]
```diff
@@ -1,9 +1,57 @@
...
+                />
+
+                <input type='submit'
+                    id='confirm-submit-button'
+                    value='Confirm'
+                />
+            </form>
         </div>
     )
 }
```
Comment (t-divya) [inline src/Header/Contact/index.js:57]: Please format the file you can remove extra spaces between elements

File: src/Header/Contact/style.css [hunk 2]
```diff
@@ -1,4 +1,49 @@
...
+    color: #fff
+}
+
+#enter-text-here-input-box {
+    padding: 8px;
+    background-color: black;
+    border: 1px solid grey;
+    border-radius: 10px;
+    width: 250px;
+    height: 200px;
```
Comment (t-divya) [inline src/Header/Contact/style.css:38]: You can decrease the height of the box a bit I think around 150 would be fine it looks bit long?

File: src/NavigationMenu/index.js [hunk 3]
```diff
@@ -11,12 +11,12 @@ export default function NavigationMenu() {
...
-                    <Route path='/' element={<Home />}></Route>
-                    <Route path = '/about' element = {<About/>}></Route>
-                    <Route path = '/project' element = {<Projects/>}></Route>
-                    <Route path= '/contact' element = {<Contact/>}></Route>
+                <Route path='/' element={<Home />}></Route>
+                <Route path='/about' element={<About />}></Route>
+                <Route path='/project' element={<Projects />}></Route>
+                <Route path='/contact' element={<Contact />}></Route>
             </Routes>
         </BrowserRouter>
```
Comment (t-divya) [inline src/NavigationMenu/index.js:22]: please format the document

**Review by t-divya — CHANGES_REQUESTED** [review]

**Review by t-divya — CHANGES_REQUESTED** [review]
changes requested

File: src/NavigationMenu/index.js [hunk 4]
```diff
@@ -11,12 +11,12 @@ export default function NavigationMenu() {
...
+            <Layout></Layout>
             <Routes>
-                    <Route path='/' element={<Home />}></Route>
-                    <Route path = '/about' element = {<About/>}></Route>
-                    <Route path = '/project' element = {<Projects/>}></Route>
-                    <Route path= '/contact' element = {<Contact/>}></Route>
+                <Route path='/' element={<Home />}></Route>
+                <Route path='/about' element={<About />}></Route>
+                <Route path='/project' element={<Projects />}></Route>
+                <Route path='/contact' element={<Contact />}></Route>
```
Comment (t-divya) [inline src/NavigationMenu/index.js:19]: spacing can be removed

**Review by t-divya — CHANGES_REQUESTED** [review]
pls check requested changes

File: src/Layout/index.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import { Outlet, Link } from "react-router-dom";
+import './style.css'
 import RCLogo from '../Image/RCLogo.png'
 
-import './style.css'
 export default function Layout() {
     return (
         <>
```
Comment (t-divya) [inline src/Layout/index.js:7]: extra space

**Review by t-divya — CHANGES_REQUESTED** [review]
extra space

**Comment by digvijayu** [comment]
image
- Button should be bigger
- placeholders missing
- Border color doesn't look the same

**Review by t-divya — APPROVED** [review]
ok looks good to me

**Pull request merged**